modules/tinymce/src/plugins/table/test/ts/browser/InsertColumnTableResizeTest.ts

```diff
@@ -1,200 +1,557 @@
+                <td style="width: 323px;"></td>
+                <td style="width: 322px;"></td>
+              </tr>
+            </tbody>
+          </table>
+        `);
+        const beforeWidth = getTableWidth(editor);
+        editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
+        editor.execCommand('mceTableInsertColAfter');
+        const afterWidth = getTableWidth(editor);
+        assert.closeTo(afterWidth, beforeWidth * 1.5, 2);
+        McEditor.remove(editor);
+      });
+
+      it('should resize table when inserting multiple columns', async () => {
+        const editor = await pCreateEditor({
+          table_sizing_mode: 'fixed',
+          table_column_resizing: 'resizetable',
+        });
+        editor.setContent(`
+          <table style="width: 674px">
+            <tbody>
+              <tr>
+                <td data-mce-selected="1" data-mce-first-selected="1" style="width: 323px;"></td>
+                <td data-mce-selected="1" data-mce-last-selected="1" style="width: 322px;"></td>
+              </tr>
+              <tr>
+                <td style="width: 323px;"></td>
+                <td style="width: 322px;"></td>
+              </tr>
+            </tbody>
+          </table>
+        `);
+        const beforeWidth = getTableWidth(editor);
+        editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
+        editor.execCommand('mceTableInsertColAfter');
+        const afterWidth = getTableWidth(editor);
+        assert.closeTo(afterWidth, beforeWidth * 2, 2);
+        McEditor.remove(editor);
+      });
+
+      it('should resize table when using a colgroup', async () => {
+        const editor = await pCreateEditor({
+          table_sizing_mode: 'fixed',
+          table_column_resizing: 'resizetable',
+        });
+        editor.setContent(`
+          <table style="width: 432px;">
+            <colgroup>
+              <col style="width: 216px;"/>
+              <col style="width: 216px;"/>
+            </colgroup>
+            <tbody>
+              <tr>
+                <td data-mce-selected="1" data-mce-first-selected="1"></td>
+                <td data-mce-selected="1" data-mce-last-selected="1"></td>
+              </tr>
+              <tr>
+                <td></td>
+                <td></td>
+              </tr>
+            </tbody>
+          </table>
+        `);
+        const beforeWidth = getTableWidth(editor);
+        editor.selection.select(editor.dom.select('td[data-mce-selected]')[0], true);
+        editor.execCommand('mceTableInsertColAfter');
+        const afterWidth = getTableWidth(editor);
+        assert.closeTo(afterWidth, beforeWidth * 2, 2);
+        McEditor.remove(editor);
+      });
+    });
+  });
+
+  context('Relative table', () => {
+    context('table_column_resizing=preservetable', () => {
+      it('should preserve table width when inserting a column', async () => {
+        const editor = await pCreateEditor({
```

Comment:
Why do we need to construct a new editor every test? From what I can see, all tests in this context use the same settings, so we should use a `TinyHook` and share the editor across all tests that are in this context.

- [ ] Don't construct an editor per test and instead construct one per "context"